templates/search/featured_search.html

```diff
@@ -11,49 +11,28 @@
     <p class="search-results__explainer--desktop">This page displays a summary of catalogue and non-catalogue content that matches your search.</p>
     <p class="search-results__explainer--mobile">Catalogue and non-catalogue content that matches your search.</p>
     <div class="featured-search__results">
-        {% with heading='Records from The National Archives' group="group:tna" %}
+        {% with results=result_groups.tna%}
             {% include './blocks/featured-search__results-block.html'%}
         {% endwith %}
-        {% with heading='Records from other UK archives' group="group:nonTna" %}
+        {% with results=result_groups.nonTna%}
```

Comment:
Tiny nitpick: It makes things a little easier to scan if there's consistant spacing between tag contents and the end of the tag (`%}`)

Reply:
Thanks @ababic - fixed now :+1: